### ISSUE TITLE
Bare Metal Installer Configure Network Interfaces

### ISSUE THREAD

**Issue by ketonik** [clearlinux/clr-installer#494]
**Describe the bug**
Installer Advanced Options - Configure Network Interfaces.
Gateway and DNS Domains fields reports in red text immediately entering a digit,
"Top level domain cannot be numeric"
actually after confusion, entering the digits and tab out of field it does actually accept digits
Same with the DNS Domain field, says it is required when actually it is not.
 
 
**To Reproduce**
Navigate to Configure Network Interfaces and select Enter.
Navigate to the network interface you wish to change.
When the desired interface is highlighted, select Enter to edit.
enter digits in Gateway field see red text complaining "Top level domain cannot be numeric"
dont enter anything in DNS Domain and in red it says required, tab out and it acctually is not required.

**Expected behavior**

**Screenshots**

![clear-linux-bare-metal-installer-interfaces](https://user-images.githubusercontent.com/49221849/63020463-c3dc9900-bed0-11e9-8c19-fefc618d7a0f.png)

**Environment (please complete the following information):**
 - Clear Linux OS Version: [swupd info]
clear-30730-live-server.iso

 - Installer Version [clr-installer -v]
2.1.1
 - Platform: [hardware infomation]
Xeon E-2146G
Supermicro X11SCZ-F 

**Additional context**

**Comment by mdhorn** [comment]
@ketonik Thanks for the feedback.

The error messages for the network fields dynamically update as you enter data into the fields.
So when you type '4', the current value is not valid. Once you fill in the field with the correct syntax, the error message goes away.

We could consider making a change, but all of the field validation in the text-based installer work this same way.  

BTW, we are planning to remove the 'Configure Network Interfaces' Advanced option in the near future as the new, preferred method is to configure the networks is to use `nmtui` or `nmcli`.

**Comment by ketonik** [comment]
Hi, i must not have made this clear enough.

I entered the first digit, immediately the red text appeared, specifically says cannot be NUMERIC assuming digits are as you explain not correct syntax. 

I entered the alpha syntax and it did not complain which justified required syntax.

However on a second attempt to test, I got back in the field deleted all characters, and entered the full Digit octet gateway.

 Again it showed red text complaining but once I tabbed out of that field the red text was removed and in fact the field DID accept numerical syntax.

![Clear-Linux-Network-Interface-Config-Success copy](https://user-images.githubusercontent.com/49221849/63152527-a9c0c900-c03e-11e9-8d0c-afe19f1d78a2.png)



Thats the most confusing and conflicting form validation I've ever filled out. Particularly painful with my host support team on the line trying to figure out why and what alpha domain to enter in place of standard gateway numerical octet they provided.

"We could consider making a change, but all of the field validation in the text-based installer work this same way."

I didn't see this conflict in any other form in the installer, it's clearly syntax misinformation of your form validation specific to the Network Interface configuration.

"BTW, we are planning to remove the 'Configure Network Interfaces' Advanced option in the near future as the new, preferred method is to configure the networks is to use nmtui or nmcli."

Anyway so when will this happen?

Personally I prefer the dialogue you have now with the forms for entry, its easy to use and it is familiar with other standard interface setups in GUIs like the Network Interface in macOS:

![Screen Shot 2019-08-16 at 15 49 32](https://user-images.githubusercontent.com/49221849/63152116-9fea9600-c03d-11e9-84c0-93d6098dac58.png)
